### PR TITLE
Create project structure

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary

_This PR is a part of the work related to issue https://github.com/ios-course/ironfoudation-team-project/issues/1._
As [discussed and agreed](https://github.com/ios-course/ironfoudation-team-project/issues/19#issuecomment-1531313031) Model-View-ViewModel (MVVM) has been chosen as the architecture pattern. 
The code will be organized by layers.

### Changes

1. Add `Model` folder.
2. Add `View` folder.
3. Add `View Model` folder.
4. Add `Helper` folder.

### CODE REVIEW

**Review by iBYarchitect — APPROVED** [review]
LGTM

**Review by iBYarchitect — CHANGES_REQUESTED** [review]
I don't see folders for Model, View Model and Helper

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
Same concerns as from @iBYarchitect 

@NadzeyaShpakouskaya make sure git stores your empty folders.
Either include some files into them or use `.gitkeep` files.

**Review by julyakmita — APPROVED** [review]
Thanks for the bug fixed! Now all  folders Model, View Model and Helper are visible.
 LGTM.

**Review by iBYarchitect — APPROVED** [review]
LGTM
nit: maybe it's good to change sitter.swift in model folder to .gitkeep?

**Review by dersim-davaod — APPROVED** [review]
LGTM

File: Woof/Woof/Model/Sitter.swift [hunk 1]
```diff
@@ -0,0 +1 @@
+import Foundation
```
[the remark this answers is not in the document]
Comment (NadzeyaShpakouskaya) [reply Woof/Woof/Model/Sitter.swift]: Fixed with [ec1a09d](https://github.com/ios-course/ironfoudation-team-project/pull/27/commits/ec1a09dae9afdd459b34e5d7c9158122f39f9148)
Replace the `Sitter` with `.gitkeep` as @iBYarchitect suggested.

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**